Directory.Build.targets

```diff
@@ -22,4 +22,16 @@
     <NETCoreAppMaximumVersion>$(MajorVersion).$(MinorVersion)</NETCoreAppMaximumVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <KnownFrameworkReference Include="Microsoft.NETCore.App"
```

Comment:
Add comment about why this is needed and open an issue in dotnet/sdk to discuss timeliness?

Reply:
https://github.com/dotnet/sdk/issues/13950